### ISSUE TITLE
Result is not UTC

### ISSUE THREAD

**Issue by zaubererty** [Realank/flutter_datetime_picker#35]
When providing a date time that is UTC you get back a date time that is UTC false.
Can you make the DateTimePicker UTC aware?

Here is my workaround:
```
onConfirm: (date) {
DateTime.utc(
  date.year, date.month, date.day,
  date.hour, date.minute,
 );
}
```

**Comment by Realank** [comment]
your workaround is perfect 😄, I know where the problem is, you can input a UTC time, and then choose a desired date, then it will return correct year/month/day/hour/sec, but UTC offset is wrong. because picker model create a new DateTime instance when change some time with current time zone. I will fix it in 1.2.0 version

**Comment by Realank** [comment]
I've fix this issue in 1.2.0, please have a try, if there is any bug, please feel free to contact with me, Thanks for your suggestion
